sqlglot/dialects/trino.py

```diff
@@ -30,5 +36,19 @@ class Generator(Presto.Generator):
             exp.JSONPathSubscript,
         }
 
+        def with_properties(self, properties: exp.Properties) -> str:
+            return self.properties(properties, wrapped=False, prefix="PROPERTIES", sep=", ")
```

Comment:
This doesn't look right, I think it breaks valid DDL statements that contain the `WITH (...)` syntax. See https://trino.io/docs/current/sql/create-table.html.

Reply:
ok let me revert.

Reply:
@georgesittas Reverted and added `check_command_warning=True`. Please take a look. Thanks

Reply:
Thanks!